custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -14,7 +14,7 @@
               media="all"
               href="{% static 'font-awesome/css/font-awesome.css' %}"/>
 
-        <link type="text/css" rel="stylesheet" media="all" href="{% static 'hqwebapp/js/lib/nvd3/nv.d3.css' %}">
+        <link type="text/css" rel="stylesheet" media="all" href="{% static 'nvd3/src/nv.d3.css' %}">
```

Comment:
Seems weird to have the css here without any JS. Are we sure this is actually used? Same for the other refs here that have this without 

FWIW I blocked this on ICDS with ublock and it seemed fine,but i'm not sure if i'm missing something

Reply:
The templates that only have the css path edited do use the js, but they're already pulling it from bower_components. I'd guess that whoever nvd3 moved to bower only grepped for the js paths.

Reply:
Ah that makes sense